tests/targets/test_android.py

```diff
@@ -154,7 +153,7 @@ def test_check_requirements(self):
         assert not hasattr(target_android, "adb_executable")
         assert not hasattr(target_android, "adb_args")
         assert not hasattr(target_android, "javac_cmd")
-        assert "PATH" not in buildozer.environ
+        del buildozer.environ["PATH"]
```

Comment:
Why `del` ?

I guess we now only need to check that into `buildozer.environ` now `PATH` is available and contains what we expect.

Reply:
I agree that:
* As suggested by the other comment, the `check_requirements` code can now be simplified to assume the presence of PATH.
* If we do that, this test code should not call del.

I will work on an update.

---

But to answer your question, my thought process was:

The old test confirmed (with an assert) that the path wasn't present (in the delta) before, and then it was present afterwards, so it had been set by `check_requirements()`.

Because it was no longer a delta, if I wanted to show that it was added by `check_requirements()`, I needed to remove it beforehand. Hence, the `del`.

Looking at the existing `check_requirements()` code, that means both versions of the test only hit one of the two code-paths, but extending the test's code-coverage of `android.py` wasn't my goal. E

(Extending the test's code-coverage of `buildozer\__init__.py` is, and in my prototypes it is up to well over 90%.)